ngsild-payloads/accountDeletionSubscription.jsonld

```diff
@@ -0,0 +1,22 @@
+{
+	"id": "urn:ngsi-ld:Subscription:AccountDeletion",
```

Comment:
others use userProfile, not account, let's be consistent. so ... `UserProfileDeletionNotification`?

Reply:
What about: `urn:ngsi-ld:Subscription:UserProfileDeletion`

Reply:
`accountDeletion` references more the general action done by the user which will trigger sending the notifcation and therefore the email (the email is to confirm his account deletion after all). `userProfileDeletion` references the entity being deleted, independantly of which action triggered the deletion.

Reply:
if we add another action to be done when an user deletes its account, how will you call it then?

Reply:
we can choose a name based on the action. And we can do the same here too. `UserProfileDeletionNotification` is just as generic as `AccountDeletion` and `userProfileDeletion `.  So maybe something like : `urn:ngsi-ld:Subscription:UserProfileDeletion:EmailNotification` or `urn:ngsi-ld:Subscription:AccountDeletion:EmailNotification` ?

Reply:
`urn:ngsi-ld:Subscription:UserProfileDeletion:EmailNotification`

(and I guess the other one should be renamed for consistency)